### PR TITLE
My Acclaro (Independent Publisher)

### PR DESCRIPTION
**Pull request by msanz-acclaro**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

### CODE REVIEW

File: independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,13061 @@
...
+                    },
+                    "emailcomment": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "emailcomment"
+                    },
+                    "quoterequired": {
+                      "type": "integer",
+                      "format": "int32",
+
```
Comment (sriyen-msft) [inline independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json]: @msanz-acclaro Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/My Acclaro/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
```
Comment (sriyen-msft) [inline independent-publisher-connectors/My Acclaro/apiProperties.json:3]: @msanz-acclaro But the connection is configured as "API Key" in the swagger, Did you test the connector with the latest files ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@msanz-acclaro Please remove unnecessary files (apiDefinition.swagger.yml, result.json, two images) from the PR.

File: independent-publisher-connectors/My Acclaro/apiProperties.json (hunk 2 above)
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/apiProperties.json]: My bad, forgot to update this file

File: independent-publisher-connectors/My Acclaro/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+## My Acclaro Connector (Independent Publisher)
+The My Acclaro client portal gives you on-demand access to the tools and reports you need to request, manage and review your translations.
+
+Easy to use and navigate, our translation management platform streamlines translation workflows for your enterprise — from order creation through review.
+
```
[the remark this answers is not in the document]
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/README.md:5]: corrected and added missing information

File: independent-publisher-connectors/My Acclaro/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## Prerequisites
+You will need the following to proceed:
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* A My Acclaro account
+
+## Obtaining Credentials 
+You need to be provided with a username and password for My Acclaro.
+
+Once you have been set up, log into the Portal and get your API key (a bearer token).
```
[the remark this answers is not in the document]
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/README.md:56]: added

File: independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,13061 @@
...
+                      "type": "string",
+                      "description": "version"
+                    },
+                    "endpoint": {
+                      "type": "string",
+                      "description": "endpoint"
+                    },
+                    "sitedescription": {
+                      "type": "string",
+                      "description": "sitedescription"
```
[the remark this answers is not in the document]
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json:55]: corrected grammatical and spell errors

**Review by msanz-acclaro — COMMENTED** [review]
all comments addressed. Please let me know if anything else is needed

File: independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,13061 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "My Acclaro",
+    "description": "My Acclaro allows you to seamlessly connect to the Acclaro client portal to submit your translation projects directly.",
+    "contact": {
+      "name": "My Acclaro Support",
+      "url": "https://www.acclaro.com/support/",
+      "email": "support@acclaro.com"
```
[the remark this answers is not in the document]
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json]: changed

File: independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json (hunk 1 above)
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json]: corrected

File: independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,13061 @@
...
+    "application/json"
+  ],
+  "paths": {
+    "/info/version": {
+      "get": {
+        "summary": "Get system information",
+        "tags": [
+          "info"
+        ],
+        "description": "Returns a variety of helpful system information including version of API source code in use,\n    service endpoint and last date of database synchronization.",
```
[the remark this answers is not in the document]
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json]: corrected

**Comment by natalie-pienkowska** [comment]
Hi @msanz-acclaro! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

**Comment by msanz-acclaro** [comment]
@sriyen-msft please accept my apologies but I don't know what else I am missing. I have addressed and resolved all conversations in this PR. The unnecessary files were removed on this commit "1596b82b14266d04aa5b47f40e884bfe044e45d0".

File: independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,13329 @@
...
+                    },
+                    "completed":{
+                      "type":"string",
+                      "description":"completed"
+                    },
+                    "canceled":{
+                      "type":"string",
+                      "description":"canceled"
+                    },
+                    "getting_qu
```
Comment (sriyen-msft) [inline independent-publisher-connectors/My Acclaro/apiDefinition.swagger.json]: @msanz-acclaro Please fix the description values everywhere in the swagger,

File: independent-publisher-connectors/My Acclaro/apiProperties.json [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        "title":"Set API Key",
+        "parameters":{
+          "x-ms-apimTemplateParameter.name":"Authorization",
+          "x-ms-apimTemplateParameter.value":"Bearer @connectionParameters('api_key')",
+          "x-ms-apimTemplateParameter.existsAction":"override",
+          "x-ms-apimTemplate-policySection":"Request"
+        }
+      }
+    ],
+    "iconBrandColor":"#f0f0f0",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/My Acclaro/apiProperties.json:30]: @msanz-acclaro You are following the template provided for IP Connectors. The default iconBrandColor for all the IP Connectors is "da3b01"

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/My Acclaro/apiProperties.json (hunk 9 above)
Comment (msanz-acclaro) [reply independent-publisher-connectors/My Acclaro/apiProperties.json:30]: @sriyen-msft I am realizing I chose the wrong path, as Acclaro (the company I work for) owns the API service, so this is not an IP Connector but a Custom connector. Please accept my apologies for the inconveniences.

**Comment by msanz-acclaro** [comment]
@sriyen-msft I am realizing I chose the wrong path, as Acclaro (the company I work for) owns the API service, so this is not an IP Connector but a Custom connector. Please accept my apologies for the inconveniences.

**Pull request closed without merging**